test/hotspot/jtreg/gc/stress/jfr/TestStressBigAllocationGCEventsWithShenandoah.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2015, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package jdk.jfr.event.gc.detailed;
+
+/**
+ * @test id=default
+ * @key randomness
+ * @requires vm.hasJFR
+ * @requires vm.gc == "Shenandoah"
```

Comment:
Use '@requires vm.gc.Shenandoah' instead. The semantics are somewhat different: '@requires vm.gc == "Shenandoah"' means 'Shenandoah has been selected at the command line when running jtreg' (which kinda implies that OpenJDK has been built with Shenandoah), which also means that this would have no effect really, while '@requires vm.gc.Shenandoah' means 'OpenJDK under test has been built with Shenandoah GC', which is really what we want: if somebody (e.g. Oracle) builds the JDK without Shenandoah, then we want to skip this test configuration, otherwise they would see failures.

Reply:
https://bugs.openjdk.org/browse/JDK-8338695